modules/dbnd-airflow/setup.py

```diff
@@ -28,7 +28,7 @@
         airflow_1_10_9=requirements_for_airflow + ["apache-airflow==1.10.9"],
         airflow_1_10_10=requirements_for_airflow + ["apache-airflow==1.10.10"],
         airflow_1_10_11=requirements_for_airflow + ["apache-airflow==1.10.11"],
-        airflow_1_10_12=requirements_for_airflow + ["apache-airflow==1.10.12"],
+        airflow_1_10_12=["apache-airflow==1.10.12"],
```

Comment:
What about other versions? Do they require the `requirements_for_airflow` necessarily?

Reply:
I think its better if we let them be. All versions lower than 1.10.12 use cattrs 0.9 and even though I tested them with py27 and they all seem to work fine I think its safer to leave them as they are and use 1.10.12.

To be honest in future we should gradually drop support for older airflow versions and port all clients to 1.10.12 and higher

Reply:
This is definitely the plan, users will be migrating to 1.10.12